crd/multitenancy/manifests/multitenancy.acn.azure.com_podnetworkinstances.yaml

```diff
@@ -71,6 +83,16 @@ spec:
                 items:
                   type: string
                 type: array
+              podNetworkStatuses:
+                additionalProperties:
+                  description: PNIStatus indicates the status of PNI
+                  enum:
+                  - Ready
+                  - CreateReservationSetError
+                  - PodNetworkNotReady
+                  - InsufficientIPAddressesOnSubnet
```

Comment:
Are you expecting us to add more for IB related here?

Reply:
We can if we need more but I don't think there is anything to add for IB as there is no real processing done for IB here, the only relevant ones would be Ready or PodNetworkNotReady, as IB doesn't create static reservations or use IP addresses